pkgs/applications/misc/systembus-notify/default.nix

```diff
@@ -17,7 +17,9 @@ let
       Type = "exec";
       ExecStart = "@out@/bin/systembus-notify";
       PrivateTmp = true;
-      ProtectHome = true;
+      # NB. We cannot `ProtectHome`, or it would block session dbus access.
+      InaccessiblePaths = "/home";
```

Comment:
Should this include `/root` as well? [The man page](https://www.freedesktop.org/software/systemd/man/255/systemd.exec.html) says that is included when `true`.

Reply:
It's a user service under `$out/lib/systemd/user`. I don't think it can access `/root` anyway.